Sources/Utilities/KMZendeskChatHandler.swift

```diff
@@ -233,5 +319,56 @@ public class KMZendeskChatHandler {
         }
         return true
     }
+    
+    func updateLastMessageCreatedTime() {
+        let channelKey = NSNumber(value: Int(groupId) ?? 0)
+        guard channelKey != 0 else {
+            print("Failed to fetch messages for group Id")
+            return
+        }
+        
+        fetchMessages(channelKey: channelKey){ messages,error,userArray in
+            guard let messageList = messages,
+                  let almessages = messageList.reversed() as? [ALMessage] else {
+                return
+            }
+            self.lastUserMessageCreatedTime = almessages.last?.createdAtTime ?? 0
+        }
+    }
+    
+    func resetLocal() {
+        isHandOffHappened = false
+        zenChatIntialized = false
+    }
+    
+    func processBufferMessages() {
```

Comment:
Are we calling it from somewhere? Also, we can pass the buffer in parameters.

Reply:
this will be called once zendesk socket connection is live.